src/components/CandidateAvatar/CandidateAvatar.scss

```diff
@@ -0,0 +1,70 @@
+.CandidateAvatar {
+    position: relative;
+    display: inline-block;
+    height: 72px;
+    width: 72px;
+
+    &__ring {
+        width: 72px;
+        height: 72px;
+    }
+
+    &__circle {
+        fill: none;
+        stroke: var(--color-neutral);
+        stroke-dasharray: 0 999;
+        stroke-linecap: round;
+        shape-rendering: geometricPrecision;
+        transition: all .3s ease;
+        transform: rotate(-90deg);
+        transform-origin: 50% 50%;
+
+        &--context {
+            @each $context in (good, warning, bad) {
+                &_#{$context} {
+                    stroke: var(--color-#{$context});
+                }
+            }
+        }
+    }
+
+    &__percentage {
+        background-color: rgba(0,0,0,.25);
+        border-radius: 50%;
+        color: white;
+        cursor: default;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        height: 100%;
+        width: 100%;
+        opacity: 0;
+        user-select: none;
+        font: {
+            family: var(--font-family-primary);
+            size: var(--font-size-large);
```

Comment:
when the image is small, this is bigger than the image. Can it be set conditionally depending on the image size?

Reply:
Can we leave this for later (if necessary)? I would have to add a class on the basis of `size`, which I think would result in the same issue as https://github.com/textkernel/nice2/pull/40#discussion_r242445265